### PR TITLE
Add "TTY: " in front of 711 <Telephone /> numbers

### PR DESCRIPTION
**Pull request by kelsonic**
## Description
**Issue:** https://github.com/department-of-veterans-affairs/va.gov-team/issues/12125

This PR adds `TTY: ` prefixed to 711 `<Telephone />` component numbers. I had mistakenly removed it because I thought the `<Telephone />` component would render it visually [due to this line](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/master/packages/formation-react/src/components/Telephone/Telephone.jsx#L37), but I actually just read that wrong lol

## Testing done
N/A

## Screenshots
N/A

## Acceptance criteria
- [x] Add back `TTY: ` for 711 numbers.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by sanlouise — APPROVED** [review]
Thanks for fixing this so fast! 💯

**Review by micahchiang — APPROVED** [review]

File: src/applications/vaos/components/NeedHelp.jsx [hunk 1]
```diff
@@ -18,13 +18,15 @@ export default function NeedHelp() {
       <p className="vads-u-margin-top--0">
         For help scheduling a VA or Community Care appointment, please call{' '}
         <a href="tel:8774705947">877-470-5947</a> (if you have hearing loss,
-        call <Telephone contact={CONTACTS['711']} pattern={PATTERNS['911']} />
+        call TTY:{' '}
+        <Telephone contact={CONTACTS['711']} pattern={PATTERNS['911']} />
```
Comment (jbalboni) [inline src/applications/vaos/components/NeedHelp.jsx:22]: Is the pattern supposed to be 911 (rather than 711)?

Comment (kelsonic) [reply src/applications/vaos/components/NeedHelp.jsx:22]: Yupp! Originally there wasn't the 711 number when the Telephone component was created just 911, which is why it's PATTERNS['911'] now. That should probably be updated in formation-react to be something like `PATTERNS.3_DIGITS` or something like that 🙂

**Review by jbalboni — APPROVED** [review]

**Review by Mottie — APPROVED** [review]

**Review by ncksllvn — APPROVED** [review]

**Review by jhonny218 — APPROVED** [review]
LGTM

**Pull request merged**